kafka-producer-application/kafka/build.gradle

```diff
@@ -29,10 +29,14 @@ repositories {
 apply plugin: "com.github.johnrengelman.shadow"
 
 dependencies {
+    implementation project(":common")
```

Comment:
probably don't need `common`? I hit an error `Project with path ':common' could not be found in root project 'kafka-producer-application'.` so just removed it. if you do need common then `settings.gradle` would need something like:
```
include ':common'
project(':common').projectDir = file('../../common')
```

Reply:
Sorry, my gradle commands assumed you were in the root project dir already. I updated the docs to reflect that and I think it should work for you from the root project dir.

Let me know if otherwise.